src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -75,13 +78,25 @@ public void parseCommand_delete_delivery() throws Exception {
                 DeleteCommand.COMMAND_WORD + " -d " + INDEX_FIRST_DELIVERY.getOneBased());
         assertEquals(new DeleteDeliveryCommand(INDEX_FIRST_DELIVERY), command);
     }
-
+    @Test
+    public void parseCommand_delete_supplier() throws Exception {
+        DeleteSupplierCommand command = (DeleteSupplierCommand) parser.parseCommand(
+                DeleteCommand.COMMAND_WORD + " -s " + INDEX_FIRST_PERSON.getOneBased());
```

Comment:
Perhaps we can add "-s" and "-d" to CLISyntax in the future for better standardisation?